modules/ob-authentication-to-container-registries.adoc

```diff
@@ -0,0 +1,47 @@
+// This module is included in the following assembly:
+//
+// builds/authenticating-build-images.adoc
+
+:_content-type: PROCEDURE
+[id="ob-authentication-to-container-registries_{context}"]
+= Authentication to container registries
+
+To push images to a private container registry, you must define a secret in the `OpenShift-Builds` namespace and then reference it in your `Build` custom resource (CR).
```

Comment:
Is the namespace definitely capitalized `OpenShift-Builds`? I thought that namespaces had to be all lowercase. There's 1 other instance of this in the PR as well.

Or is this more of a general reference to "OpenShift Builds", and then just whatever namespace you are working in? I see below in the procedure that the `create secret` command mentions a `--namespace <namespace>`, but doesn't comment on what the namespace should be.

Reply:
@apoorvajagtap, This one as well^^ Is the casing of the `OpenShift-Builds` namespace correct? Also, is it mandatory for a user to create a secret in this namespace only. If yes, then we might need to replace `oc --namespace <namespace>` with `oc --namespace OpenShift-Builds` in the commmands.

Reply:
@shipsing these secrets are expected to be available in the respective namespaces. (ref: [upstream doc](https://shipwright.io/docs/build/authentication/#authentication-to-container-registries))
I think we should replace the `OpenShift-Builds` with `respective` here.